tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+
+- block:
+
+    - import_tasks: secret-token.yml
+
+    - name: "Copy Helm config to host"
+      template:
+        src: "{{ item }}.yaml.j2"
+        dest: "/root/helm/{{ item }}.yaml"
+        mode: 0600
+      with_items:
+        - jupyter_config
+        - secret_config
+      register: config
+
+  when: pangeo_state == "present"
+
+- name: Check if the chart exists
+  command: "helm ls {{ pangeo_namespace }} --output json"
+  register: output
+  changed_when: false
+
+- name: Set fact about the output
+  set_fact:
+    current_release: "{{ ({} if output.stdout == '' else (output.stdout | from_json)).get('Releases', [{}]) | first }}"
```

Comment:
Could this be made cleaner with the `json_query` filter?

https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#json-query-filter